internal/builders/bazel/rebuilder.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/bash
+#
+# Copyright 2023 SLSA Authors
```

Comment:
can you separate code that builds provenance, vs code that is utilities (color, etc)? I think we need 2 files.